tools/asynq/cmd/root.go

```diff
@@ -402,11 +405,16 @@ func getRedisConnOpt() asynq.RedisConnOpt {
 }
 
 func getTLSConfig() *tls.Config {
+	if viper.GetBool("tls") {
+		return &tls.Config{InsecureSkipVerify: viper.GetBool("insecure")}
```

Comment:
Looks a bit confusing to users who might want to use both and then it returns early on --tls and then it can't connect in instances where the hostname doesn't match.